index.js

```diff
@@ -1,3 +1,4 @@
+require('babel-plugin-check-es2015-constants');
```

Comment:
What is this line trying to do? Right now it does nothing, because it does not get included in `module.exports.plugins`.